src/managed/dir.proj

```diff
@@ -16,6 +16,7 @@
       <BuildArgs>/p:Configuration=$(ConfigurationGroup)</BuildArgs>
       <BuildArgs Condition="'$(LatestCommit)' != ''">$(BuildArgs) /p:LatestCommit=$(LatestCommit)</BuildArgs>
       <BuildArgs Condition="'$(BuiltByString)' != ''">$(BuildArgs) /p:BuiltByString="$(BuiltByString)"</BuildArgs>
+      <BuildArgs>$(BuildArgs) /p:BaseOutputPath=$(BaseOutputRootPath)</BuildArgs>
```

Comment:
Currently no, dir.props brings in too much stuff, and IntermediateOutputPath depends on a bunch of derived properties which happen in dir.props, so we just have to pass it explicitly.  As mentioned in the description, if dir.props gets refactored and we can pull in just the piece that we want, then we can add that import to the common props file.